ChangeLog

```diff
@@ -14,6 +24,7 @@
 	* man/BermudanSwaption.Rmd: Modified to reflect DiscountCurve
 	option and example updated
 	
+>>>>>>> eddelbuettel/master
```

Comment:
See above.

You still don't know how to update your repo correctly to not have merge conflicts.  That is a little worrysome.